### PR TITLE
docs: Update readme

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
* Add version 6.0 to README
* Added needed dynatrace.conf.yaml to the `keptnConfigureMonitoring` example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,22 +8,23 @@ Jenkins shared library for integrating Keptn Use Cases within your Jenkins Pipel
...
+| 3.1             | 0.7.x         | Sending *buildId* label to Keptn                                  |
+| 3.2             | 0.7.x         | Adding custom label support for Keptn 0.7.x                       |
+| 3.3             | 0.7.x         | Improved Evaluation done event handling in Keptn                  |
+| 3.4             | 0.7.x         | Implementing #9 to customize image and tag                        |
+| 3.5             | 0.7.x         | Keptn API Token now configurable via Jenkins Credentials          |
+| 4.0             | 0.8.0         | Now supporting Keptn 0.8.0                                        |
+| 4.1             | 0.8.x, 0.9.x  | Supporting Keptn 0.9.x, bug fixes                                 |
+| 5.0             | 0.9.x, 0.10.0 | Supporting Keptn 0.10.0, bug fixes, Cleanups                      |
+| 5.1             | 0.9.x, 0.10.0 | Bug fixes                                                         |
+| 6.0 *pending*   | 0.10.0 - 0.12.0 | Supporting Keptn 0.12.0, added `keptnConfigureMonitoring` command |
```
Comment (TannerGabriel) [inline README.md:27]: I guess "pending" will be removed after the 6.0 release has been created?

**Review by TannerGabriel — COMMENTED** [review]
I also noticed that the heading "Pre-Requisites" is misspelled.

File: README.md (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply README.md:27]: that's the plan. a pre-release is currently available

**Comment by christian-kreuzberger-dtx** [comment]
Fixed typo "Requesit"

**Pull request merged**